crypto/yubihsm.go

```diff
@@ -162,8 +162,10 @@ func (privKey *YubiHsmPrivateKey) deletePrivKey() {
 	}
 
 	// send command
-	privKey.sessionMgr.SendEncryptedCommand(command)
-
+	_,err =privKey.sessionMgr.SendEncryptedCommand(command)
+    if err != nil{
```

Comment:
this doesn't look correct, lets just log if there is an error it is not going to remove the privKeyID

Reply:
Done.